.github/workflows/test.yml

```diff
@@ -90,12 +90,307 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
-  # Run all the zebra tests, including tests that are ignored by default
-  test-all:
-    name: Test all
+  # # Run all the zebra tests, including tests that are ignored by default
```

Comment:
Blocker: Most of our tests will be disabled if we merge this PR.

This seems like a left over development hack.

Reply:
Yes, this was a development hack so we didn't had to wait for the whole process. If we're done with testing the timing, then I'll finalize this PR

Reply:
I've tried all the quick tweaks I could think of.

If we want more speedups, we'll need to do analysis like:
- #3375

Reply:
I think we can do that afterwards. From my PoV there's more benefit on merging this PR ASAP and solving issues in the road than actually waiting for 30 minutes or 1h less in the test.

Reply:
Yes, I agree.